client/components/deposits-overview/recent-deposits-list.tsx

```diff
@@ -0,0 +1,112 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { Flex, FlexItem, Icon } from '@wordpress/components';
+import { calendar } from '@wordpress/icons';
+import { Link } from '@woocommerce/components';
+
+/**
+ * Internal dependencies.
+ */
+import strings from './strings';
+import './style.scss';
+import DepositStatusChip from 'components/deposit-status-chip';
+import { getDepositDate } from 'deposits/utils';
+import { CachedDeposit } from 'wcpay/types/deposits';
+import { useDeposits } from 'wcpay/data';
+import { formatCurrency } from 'wcpay/utils/currency';
+import { getDetailsURL } from 'wcpay/components/details-link';
+
+interface DepositRowProps {
+	deposit: CachedDeposit;
+}
+interface RecentDepositsList {
+	deposits: CachedDeposit[];
+	isLoading: boolean;
+}
+interface RecentDepositsProps {
+	currency: string | undefined;
+}
+
+const tableClass = 'wcpay-deposits-overview__table';
+
+const useRecentDeposits = ( currency?: string ): RecentDepositsList => {
```

Comment:
Not really 100% certain on this, so maybe @Jinksi can confirm, but should we move this `useRecentDeposits` hook into a `hooks.ts`?
I noticed this is what we did in the account-balances component

Reply:
I don't have a strong reason for or against including in a new `hooks.ts` file, since this hook is only used within this file.

However, after a quick search of WCPay shows, it looks like including hooks in `hooks.ts` is a more common practice.

Reply:
I think a dedicated, explicit hooks file is good practice. Encourages decoupling and thinking about hook interface in isolation.